src/nls/root/strings.js

```diff
@@ -389,7 +389,7 @@ define({
     "CMD_WORKING_SORT_TOGGLE_AUTO"        : "Automatic Sort",
     "CMD_THEMES"                          : "Themes\u2026",
 
-    // Navigate menu Commands
+    // Navigate menu commands
```

Comment:
I believe it is because of the module name `command/Commands` which is traditionally loaded to `Commands` variable. :) It's OK though to have it either way, does not matter.